src/platform/forms-system/test/js/review/ReviewPage.unit.spec.jsx

```diff
@@ -55,11 +55,26 @@ describe('Schemaform review: ReviewPage', () => {
         location={location}
       />,
     );
-
     expect(tree.find('withRouter(Connect(ReviewChapters))')).to.have.length(1);
     expect(tree.find('withRouter(Connect(SubmitController))')).to.have.length(
       1,
     );
     tree.unmount();
   });
+
+  it('should appropriately render a downtime notification', () => {
+    const tree = shallow(
+      <ReviewPage
+        form={form}
+        openChapters={[]}
+        route={{ formConfig, pageList }}
+        setEditMode={f => f}
+        setPreSubmit={f => f}
+        location={location}
+      />,
+    );
+
+    expect(tree.find('Connect(DowntimeNotification)')).to.have.length(1);
```

Comment:
I'm not proud of this test :grimacing:

Reply:
Hehehe, I think it's fine. I'd give you bonus points for ensuring that we're passing the right props, but any more than that and we're getting into the realm of testing the `DowntimeNotification` component itself.